src/SFA.DAS.ApprenticeCommitments.Web/Pages/Apprenticeships/TransactionComplete.cshtml.cs

```diff
@@ -28,5 +28,10 @@ public async Task OnGet()
             var apprenticeship = await _client.GetApprenticeship(ApprenticeshipId.Id);
             CourseName = apprenticeship.CourseName;
         }
+
+        public async Task<IActionResult> OnPostSurvey()
+        {
+            return new RedirectResult("https://forms.office.com/Pages/ResponsePage.aspx?id=yXfS-grGoU2187O4s0qC-XVnVtImmRBOiQ5Eo7TmxhBURDVJOFZSSDNIM05JUlpTWkwzWEVXOEE5Ny4u");
```

Comment:
This might be good to be in a configuration in the longer term, but we can live with it for now